google-account-plugin/src/com/google/cloud/tools/intellij/login/IntellijGoogleLoginService.java

```diff
@@ -169,6 +169,7 @@ public void logIn(@Nullable final String message,
       @Nullable final IGoogleLoginCompletedCallback callback) {
     UsageTrackerProvider
         .getInstance().trackEvent(LoginTracking.CATEGORY, LoginTracking.LOGIN, "login.start", null);
+    final CredentialedUser activeUser = users.getActiveUser();
     users.removeActiveUser();
```

Comment:
Just a question: I was wondering what will happen if we just don't call `removeActiveUser()`.